templates/files/manifests/controller-manager.yaml

```diff
@@ -68,4 +71,7 @@ spec:
     name: k8s-kubeconfig
   - hostPath:
       path: /etc/kubernetes/secrets
-    name: k8s-secrets
\ No newline at end of file
+    name: k8s-secrets
+  - hostPath:
+      path: /var/libexec/kubernetes/kubelet-plugins/volume/exec
```

Comment:
I would still rather have this under `/var/lib/kubelet/plugins/volume/exec` as that is the place where rest of kubelet files are and its also separate volume than root volume